ngraph/test/files/paddlepaddle/models/models.csv

```diff
@@ -25,6 +25,7 @@ bilinear_upsample_false_1,
 bilinear_upsample_scales,
 bilinear_upsample_scales2,
 bilinear_upsample_true_0,
+bmm,
```

Comment:
I still see that some generated models are missing in this list, like 'bilinear_upsample_tensor_size'. But I don't have a script to check this automatically, I'm just looking on the list

Reply:
@nosovmik the the missing interpolate models conversion are test in another test because IE cannot inference with dynamic shape. You can find them at ngraph_dev/dev_openvino/ngraph/test/frontend/paddlepaddle/basic_api.cpp. For other test cases, I will check with my colleagues.

Reply:
@ceciliapeng2021 @luo-cheng2021 Please fix missing test cases

Reply:
@nosovmik @zhangYiIntel There are 2 test cases for pool2d with NHWC layout. They are disabled for now as the feature will be enabled in CVS-55483.

Reply:
@ceciliapeng2011 Maybe still add them but commented out, e.g. #pool2d_<...>

Reply:
Add comments for all currently disabled test in f01825751710bd270ccea377b4499c4957cf8a3d